rails/Gemfile

```diff
@@ -0,0 +1,10 @@
+source 'https://rubygems.org'
+
+gem 'benchmark-ips', github: 'kirs/benchmark-ips', branch: 'quiet-mode'
```

Comment:
what is the value of quiet mode? how do you access results?

On Sat, Mar 14, 2015 at 8:46 PM, Kir Shatrov <notifications@github.com>
wrote:

> In rails/Gemfile
> <https://github.com/ruby-bench/ruby-bench-suite/pull/8#discussion_r26436713>
> :
>
> > @@ -0,0 +1,10 @@
> > +source 'https://rubygems.org'
> > +
> > +gem 'benchmark-ips', github: 'kirs/benchmark-ips', branch: 'quiet-mode'
>
> Mine has quiet mode feature, but I've already sent a PR:
> evanphx/benchmark-ips#36
> <https://github.com/evanphx/benchmark-ips/pull/36>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ruby-bench/ruby-bench-suite/pull/8/files#r26436713>.
>